cmd/genutils/genutils.go

```diff
@@ -18,10 +18,25 @@ package genutils
 
 import (
 	"fmt"
+	"io"
 	"os"
 	"path/filepath"
 )
 
+type devNull int
```

Comment:
If we added this type, it should be a `struct {}`, and the type should be exported (rather than a global variable). But I don't think we should add this, see my later comment.